helpers/templates/time.go

```diff
@@ -41,8 +46,20 @@ func convertToFloat(i interface{}) (float64, error) {
 	}
 }
 
+func FloatToTime(v float64) (*time.Time, error) {
```

Comment:
In prometheus/prometheus this is unexported. I'm probably lacking context, but any reason to export it here?

Reply:
My reasoning was the same as ConvertToFloat: we can remove it from prometheus repo and reuse this implementation in both alertmanager and prometheus repos

Reply:
yeah it's also used in `toTime`:
```
			"toTime": func(i interface{}) (*time.Time, error) {
				v, err := convertToFloat(i)
				if err != nil {
					return nil, err
				}

				return floatToTime(v)
			},
```

that way (by exporting it) I can just remove the implementation from the prometheus repo and use the common one

Reply:
perfect